pages/register.vue

```diff
@@ -9,7 +9,7 @@
       />
       <v-btn
         @click="register"
-        color="primary"
+        class="primary-light-1-background"
```

Comment:
Certain things I want to have control over both the text color and the background color independently, using the color prop doesn't allow for that flexibility. This'll still change with the theme because those values in the class are set to theme vars

Reply:
ah yeah v-btn kinda weird so where the color is used depends on the variants maybe one of these accomplishes what you want using `color` ? https://vuetifyjs.com/en/components/buttons/#variants